### PR TITLE
Added Modular and bindable date parser + bug on parsing date on tab keypress 

### PR DESCRIPTION
**Pull request by Kukks**
https://github.com/aurelia-ui-toolkits/aurelia-materialize-bridge/issues/197

from the updated samples doc:
    If you would like to configure different parsers to handle the editable option, simply bind the parsers property and push/remove/replace the array with your own Parser classes. The Parser class would need to consists of canParse(value) and parse(value). The example here allows you to type in "yesterday" and will parse for you yesterday's date.

### CODE REVIEW

File: sample/src/samples/datepicker/advanced-options.js [hunk 1]
```diff
@@ -1,3 +1,7 @@
+import {DatePickerDefaultParser} from '../../../../src/aurelia-materialize-bridge/datepicker/datepicker.default-parser';
```
Comment (Thanood) [inline sample/src/samples/datepicker/advanced-options.js]: Why not `import {DatePickerDefaultParser} from 'aurelia-materialize-bridge/datepicker/datepicker.default-parser';`? Did that not work?

**Comment by Thanood** [comment]
Very nice, thanks!
Would you mind not pushing the dist folder? I know you work from a fork and import that directly into your project but maybe it's possible.

**Comment by Kukks** [comment]
Hmm sure, But it will have to be from the next time since I pushed it all in one commit this time. I'll start doing feature branches then :)

File: sample/src/samples/datepicker/advanced-options.js (hunk 1 above)
Comment (Kukks) [inline sample/src/samples/datepicker/advanced-options.js]: ah, did not come to mind. I didn't check if there was a aurelia-materialize-bridge alias in the config.js for the samples so I played it safe and did it directly ;) 
I can change tomorrow though :)

**Comment by Thanood** [comment]
When I set a max date, I can still enter something out of range and the reult is stored in the view model. The date picker correctly restricts the date..

**Comment by Thanood** [comment]
Okay, fixed that. :smile:

**Pull request merged**